packages/hash/shared/src/types.ts

```diff
@@ -1,393 +1,8 @@
-import { BaseUri, VersionedUri } from "@blockprotocol/type-system";
-import slugify from "slugify";
-import { systemUserShortname } from "@hashintel/hash-shared/environment";
-import { frontendUrl } from "./environment";
+// The types exported here are expected to be branded.
 
-type SchemaKind = "data-type" | "property-type" | "entity-type";
+export type Uuid = string;
+export type AccountId = Uuid;
+export type ownedById = Uuid;
+export type updatedById = Uuid;
```

Comment:
These should be capitalised, right?

Reply:
Yes, you're right 98f9b73f8af33f98478c3f2ba80e81979780ab23